.env.example

```diff
@@ -21,3 +21,21 @@ NEST_PORT
 JWT_SECRET
 # Expects a (value-name) string, eg.: '1w', '3600s'
 JWT_EXPIRE
+
+# Expects a valid email service, eg.: 'gmail'
+EMAIL_SERVICE
```

Comment:
What are other possible options? SMTP? Could you provide the package docs for this value?

Reply:
Also, can you set an SMTP port and host then?

Reply:
https://community.nodemailer.com/2-0-0-beta/setup-smtp/well-known-services/

Possibly, didn't try that.